src/pwr.ps1

```diff
@@ -727,8 +727,25 @@ if (-not $Run) {
 	}
 }
 
+function Get-PwrConfig {
+	$PwrCfgDir = $ExecutionContext.SessionState.Path.CurrentLocation
+	do {
+		$PwrCfg = "$PwrCfgDir\pwr.json"
+		if (Test-Path -Path $PwrCfg -PathType Leaf) {
+			try {
+				return Get-Content $PwrCfg -ErrorAction 'SilentlyContinue' | ConvertFrom-Json
+			} catch {
+				Write-PwrFatal "pwr: bad JSON parse of file $PwrCfg - $_"
+			}
+		}
+		$PwrCfgDir = Split-Path -Parent $PwrCfgDir
+	} while ($PwrCfgDir -ne $null)
```

Comment:
`$null should be on the left side of equality comparisons.PSScriptAnalyzer(PSPossibleIncorrectComparisonWithNull)`